<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaAlterarMarca</class>
 <widget class="QDialog" name="TelaAlterarMarca">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_AlterarMarca">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Alterar</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/img/devices_floppydisk_dispositivos_691.png</normaloff>:/rec/img/devices_floppydisk_dispositivos_691.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_img">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_TituloMarca">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Voltar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-icon:url(:/rec/img/reply-arrow-filled.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Digite o nome da marca:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_InserirFoto">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserir Foto</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_IDMarca">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID Marca</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Buscar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_IDMarca</tabstop>
  <tabstop>lineEdit_TituloMarca</tabstop>
  <tabstop>pushButton_InserirFoto</tabstop>
  <tabstop>pushButton_AlterarMarca</tabstop>
  <tabstop>pushButton_Voltar</tabstop>
  <tabstop>pushButton_Buscar</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
